src/script/ceph-release-notes

```diff
@@ -104,9 +104,9 @@ def split_component(title, gh, number):
         issue_labels = {it['name'] for it in issue['labels']}
         if 'documentation' in issue_labels:
             return 'doc: ' + title
-        item = labels.intersection(issue_labels)
+        item = (labels.union(set(prefixes))).intersection(issue_labels)
```

Comment:
Hm I looked at this wrong, what I wanted is if the pr is not labeled and there is a prefix that matches then we pass instead of returning unknown which we're doing now, this requires one more modification, I'll look into it.